src/worker.cc

```diff
@@ -77,7 +77,11 @@ void Worker::newConnection(evconnlistener *listener, evutil_socket_t fd,
     return;
   }
   event_base *base = evconnlistener_get_base(listener);
+#ifndef DISABLE_USE_PTHREADS
   auto evThreadSafeFlags = BEV_OPT_THREADSAFE | BEV_OPT_DEFER_CALLBACKS | BEV_OPT_UNLOCK_CALLBACKS;
```

Comment:
@git-hulk I am not  familiar with `BEV_OPT_THREADSAFE`, AFAIK, one thread may operate another thread's event buffer, so, is it ok?

Reply:
https://github.com/bitleak/kvrocks/pull/94 (FIX: add BEV_OPT_THREADSAFE option (monitor cmd will coredump when evbuffer_add and evbuffer_drain concurrently in multi thread))